webpack/scenes/Settings/Tables/__tests__/TableReducer.test.js

```diff
@@ -0,0 +1,49 @@
+import * as types from '../TableConstants';
+
+import {
+  initialState,
+  loadingState,
+  requestSuccessResponse,
+  successState,
+  tableRecord,
+  errorState,
+} from './Table.fixtures';
+import reducer from '../TableReducer';
+
+describe('tables reducer', () => {
```

Comment:
You could write much simpler test here if you would use the test-helper `testReducerSnapshotWithFixtures`.
https://github.com/Katello/katello/blob/master/webpack/move_to_pf/test-utils/testHelpers.js#L67

We still don't have an example in katello but you can see an example in foreman:
https://github.com/theforeman/foreman/blob/c28d51eb132cc079dd7a5ad7fbef63aba3cca2ab/webpack/assets/javascripts/react_app/components/PasswordStrength/__tests__/PasswordStrengthReducer.test.js

If you want to, it could be nice to use the `testReducerSnapshotWithFixtures`.

Reply:
Went ahead and refactored the test.  Thank you for pointing me to the helpers.